pysrc/bytewax/tracing.py

```diff
@@ -0,0 +1,42 @@
+"""
+Tracing and logging in bytewax are handled in the rust side,
+to offer a really detailed view of what is happening in your dataflow.
+
+By default, bytewax sends all "error" logs to the standard output.
+This can be configured with an environment variable, "BYTEWAX_LOG".
```

Comment:
Currently the env var is only used for logging to console, not for configuring tracing.

I opted for this since logging is usually configurable this way, so that when a dataflow is deployed, you can change the logging level without touching the code. Thinking about it, in our case this problem is mitigated by the fact that you usually don't need to rebuild a container if you change the code, and also using something like `proc_env` would still allow users to do that through env var.

I'll work on this to make it more consistent with the rest of the config options

Reply:
Ok, I made the `log_level` a kwarg for `run_main` and `spawn_cluster` and `cluster_main`, and added the option to retrieve it in the parse module, let me know if this is what you meant.